src/closure_externs.js

```diff
@@ -77,13 +77,3 @@ var PromiseConstructorLike;
 
 /** @typedef {?} */
 var SymbolConstructor;
-
-/**
- * This type is used as a marker in locations where TypeScript knows the type
- * but we cannot express that type in the Closure system.
- *
- * It's used in 'implements' clauses so that a person debugging the emitted JS
- * can see that tsickle saw the type but ignored it.
- * @record
- */
-class InexpressibleType {}
```

Comment:
I think one of your CLs used InexpressibleType in its emit, you might need to retain this part?  (If 'git grep InexpressibleType' in your tree after this patch has no matches, then ignore this.)